src/command/mcl.zig

```diff
@@ -11,6 +11,23 @@ var line_accelerations: []u7 = undefined;
 const Direction = mcl.Direction;
 const Station = mcl.Station;
 
+const RegisterLogging = struct {
+    line_index: u8,
```

Comment:
Ideally, use `mcl.Line.Index`, which lets you change the field name to just `line`